dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -309,7 +309,7 @@ def run_validation(self, job):
 
             error_list.write_all_row_errors(job_id)
             # Update error info for submission
-            populate_submission_error_info(submission_id)
```

Comment:
If we take this function out of this line, do we call it anywhere else other than in tests? If not, can we just cut the entire function? (submission error info is still populated with direct calls to `sum_number_of_errors_for_job_list` right?)